### ISSUE TITLE
Adding Telegram icon in Social Menu

### ISSUE THREAD

**Issue by alvarosaco** [kovshenin/semicolon#1]
Hi,

How could I add a Telegram icon in Social Menu?

View: - https://github.com/Automattic/Genericons/issues/54

Thanks.

**Comment by alvarosaco** [comment]
I have solved it adding Font-Awesome to style.css file:

`@import url(https://netdna.bootstrapcdn.com/font-awesome/4.6.1/css/font-awesome.css);`

and creating the rule:

```
ul.semicolon-social li a[href*="telegram.me"]::before {
    font-family: 'FontAwesome';
    content: "\f1d8"; // fa-paper-plane
}
```

View example: http://elentendedero.es

**Comment by kovshenin** [comment]
Fixed in f554f40ceea7d29e3188d7eff9bc23f3ce52e1cc
